.github/workflows/ci-prb.yml

```diff
@@ -35,7 +35,8 @@ jobs:
         env:
           CI: true
           JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8
-        run: ./gradlew --parallel clean test
+        run: |
+            sudo env "PATH=$PATH" bash -c "ulimit -l 65536 && ulimit -a && ./gradlew --no-daemon --parallel clean test"
```

Comment:
we want to enable CI on other platforms (windows, macOS). can the `ulimit` and related changes be done based upon OS (see `if: ${{ runner.os == 'Windows' }}` above)?

Reply:
I added 'gradle-command' to the build matrix.  If macOS is added to the matrix in the future, you can easily set a different Gradle build command.